K8s docs: Manual join: add note that kubeconfig not required (#9998)
hashicorp/consul: website/content/docs/k8s/installation/deployment-configurations/clients-outside-kubernetes.mdx

If you are unable to use auto-join, you can also follow the instructions in
either of the auto-join sections but instead of using a `provider` key in the
`-retry-join` flag, you would need to pass the address of at least one

> consul server, e.g: `-retry-join=$CONSUL_SERVER_IP:$SERVER_SERFLAN_PORT`. Also,
> a `kubeconfig` file is not required for manual join. 

Reviewer: ```suggestion
consul server, e.g: `-retry-join=$CONSUL_SERVER_IP:$SERVER_SERFLAN_PORT`. A
`kubeconfig` file is not required when using manual join.
```
